### PR TITLE
Various Dark Theme Fixes

### PR DESCRIPTION
**Pull request by Roguim**
- `DARK-5`
- `DARK-8`
- `DARK-10`
- `DARK-16`
- `DARK-26`
- `DARK-41`

### CODE REVIEW

File: css/modern.css [hunk 1]
```diff
@@ -316,6 +316,22 @@ footer button,
     color: var(--contrast-text) !important;
 }
 
+
+.action-links-wrapper-expanded .action-links-unfold .action-links-unfold-text {
+    background-color:var(--text) !important;
```
Comment (aopell) [inline css/modern.css:321]: I'm not entirely certain what section this is for, but why is the background color a text color?

File: css/modern.css [hunk 2]
```diff
@@ -690,6 +734,23 @@ img.splus-logo-inline {
     color: var(--text) !important;
 }
 
+#edge-filters-btn {
+    background-color: var(--text) !important;
+    color: var(--secondary) !important;
```
Comment (aopell) [inline css/modern.css:739]: Just making sure again that you meant to use a text color for the background and a background color for the text

File: css/modern.css [hunk 3]
```diff
@@ -905,6 +966,37 @@ body #right-column-inner a.link-btn {
...
+
+.sgy-course-search-form__submit-row {
+    background-color: transparent !important;
+    border-color: transparent !important;
+}
+.sgy-react-select .Select-control, .sgy-react-select .Select-menu-outer {
+    color: var(--text) !important;
+    border-color: transparent !important;
+    border-radius: var(--border-radius) !important;
+}
```
Comment (aopell) [inline css/modern.css:979]: A few more borders and backgrounds that are transparent. Can we use `none` instead?

File: css/modern.css [hunk 4]
```diff
@@ -1528,6 +1620,16 @@ div.cellphone-container {
...
 
 
+.settings-table tbody td.selected, .settings-table tbody td.selected:hover {
+    background-color: var(--active) !important;
+}
+.settings-table tbody td {
+    background-color: var(--primary) !important;
+}
+.settings-table tbody td.realm-disabled, .settings-table tbody td.realm-disabled:hover, .settings-table tbody td.area-disabled {
+    background-color: transparent !important;
```
Comment (aopell) [inline css/modern.css:1630]: Transparent -> none?

**Review by aopell — CHANGES_REQUESTED** [review]
See comments scattered throughout. Not necessarily saying any of this needs to be changed, just want some understanding of your thoughts on these.

File: css/modern.css (hunk 4 above)
Comment (Roguim) [reply css/modern.css:1630]: No idea if transparent is any different to none.

File: css/modern.css (hunk 2 above)
Comment (Roguim) [reply css/modern.css:739]: Yeah I did cause it's inverted

File: css/modern.css [hunk 5]
```diff
@@ -316,6 +316,22 @@ footer button,
     color: var(--contrast-text) !important;
 }
 
+
+.action-links-wrapper-expanded .action-links-unfold .action-links-unfold-text {
+    background-color:var(--text) !important;
+    color:var(--secondary) !important;
+    filter: invert(1) !important;
```
[the remark this answers is not in the document]
Comment (Roguim) [reply css/modern.css:323]: It should be invert(1), I believe I tested on light theme and it worked fine

**Review by aopell — CHANGES_REQUESTED** [review]
Will you check the resources page? Everything has turned white for me when I test out your PR. I think you may have accidentally changed something to remove its background colors. Also, the dropdown menus are white when opened.
image

**Comment by Roguim** [comment]
> Will you check the resources page? Everything has turned white for me when I test out your PR. I think you may have accidentally changed something to remove its background colors. Also, the dropdown menus are white when opened.
> 
> image
> 
> Also DARK-16 doesn't seem fixed to me
> image

Which browser are you using? Mine works fine on Brave and Firefox when I tested it, I want to see where it doesn't work so I can fix it.

**Comment by aopell** [comment]
> > Will you check the resources page? Everything has turned white for me when I test out your PR. I think you may have accidentally changed something to remove its background colors. Also, the dropdown menus are white when opened.
> > image
> > Also DARK-16 doesn't seem fixed to me
> > image
> 
> Which browser are you using? Mine works fine on Brave and Firefox when I tested it, I want to see where it doesn't work so I can fix it.

It's not working on Chrome for me

**Comment by Roguim** [comment]
> It's not working on Chrome or Firefox for me

Which version? I'll install it and test ^^
Alternatively, it might only be an issue on LAUSD schoology, which I don't have access to.

**Comment by aopell** [comment]
Can you just fix the conflicts now and then I'll merge!

**Comment by Roguim** [comment]
> Can you just fix the conflicts now and then I'll merge!

Oh ok, sure, I was trying to work out how to fix the bug in firefox though

**Comment by Roguim** [comment]
Resolved conflicts and FF bug should be fixed now.

**Review by aopell — APPROVED** [review]

**Pull request merged**